docs/index.html

```diff
@@ -23,7 +23,7 @@
                 <span class="icon-bar"></span>
             </button>
             <a class="navbar-brand" href="https://github.com/thesabbir/simple-line-icons">Simple Line Icons </a>
-            <span class="version">2.5.0</span>
+            <span class="version">2.5.2</span>
```

Comment:
I didn't intentionally make this change, but when I ran the `build` script this got updated. It seems to be an appropriate change though?